lib/pipely/deploy/client.rb

```diff
@@ -47,9 +47,10 @@ def deploy_pipeline(pipeline_basename, definition)
         @log.info("#{pipeline_ids.count} existing pipelines: #{pipeline_ids}")
 
         # Create new pipeline
-        created_pipeline_id = create_pipeline(pipeline_name,
-                                              definition,
-                                              tags)
+        created_pipeline_id = create_pipeline(
+          pipeline_name, tags, &block
```

Comment:
@dstuebe - You don't need to specify a block argument in order for a method to yield.
http://www.reactive.io/tips/2008/12/21/understanding-ruby-blocks-procs-and-lambdas/

Specifying the block argument would let you dereference the block into a Proc, but Proc#call is slower and clunkier than simply yielding to the block.
http://stackoverflow.com/questions/1410160/ruby-proccall-vs-yield